gizmo.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+import numpy as np
+import pandas as pd
+
+def hello(name, country='Finland'):
+    
+    print('Hello {}, how are things in {}?'.format(name,country))
+    
+    return
+
+
+def spell(str_in):
+    
+    str_out = str_in[0]
+    for el in str_in[1:]:
+        str_out += '.' + el
+    print(str_out)
+    
+    return
+
+
+def relative_path(subject_identifiers):
+    
+    rel_path = './subjects/mock_recording_'
+    ext = '.rec'
+    
+    file_list = [rel_path + elem + ext for elem in subject_identifiers]
+    
+    return file_list
+
+
+class Gizmo(object):
+  
+    def __init__(self, name):
+      
+        self.name = name
+      
+        return
+    
+    def speak(self):
+        
+        print(self.name)
+        
+        return
```

Comment:
why do you end each function with `return`?

Reply:
I am used to close/finish functions with something (from Fortran and C I suppose) and I was under the impression that, as a safeguard, it was good practice to finish the function with `return`.

Reply:
it's unusual in Python, but I kind of like it.